src/app/app.component.html

```diff
@@ -184,40 +201,26 @@
                         (globalActionEvent)="getBoardEvents({ origin: list.identifier, event: 'globalActionEvent', data: $event })"
                         (changeResultMode)="changeListResultMode($event,list.identifier)">
                       </arlas-result-list>
-
                     </div>
                   </mat-tab>
                 </mat-tab-group>
               </div>
             </div>
 
+            <!-- TODO: what ? -->
```

Comment:
This button was designed to refresh analytics board data whenever you want.
It was useful when using the tagger, to avoid having to refresh the page as the tagging process progressed.

Reply:
Okay, I will integrate it in the left panel below the analytics then